src/cswinrt/main.cpp

```diff
@@ -35,7 +35,7 @@ namespace cswinrt
         { "target", 0, 1, "<net6.0|net5.0|netstandard2.0>", "Target TFM for projection. Omit for compatibility with newest TFM (net5.0)." },
         { "component", 0, 0, {}, "Generate component projection." },
         { "verbose", 0, 0, {}, "Show detailed progress information" },
-        { "embedded", 0, 0, {}, "Generate the projection as internal."},
+        { "internal", 0, 0, {}, "Generate the projection as internal."},
```

Comment:
We should support both internal and embedded or go back to embedded to avoid breaking existing customers.